tests/cypress/views/savedSearches.js

```diff
@@ -19,17 +19,17 @@ export const savedSearches = {
     searchBar.whenEnterTextInSearchBar('kind', 'cluster')
     searchBar.whenEnterTextInSearchBar('ManagedClusterJoined', 'True')
 
-    cy.get('.pf-c-expandable-section__toggle', { timeout: 6000 }).then(
+    cy.get('.pf-c-expandable-section__toggle').then(
       ($btn) => {
         var fullText = $btn.text()
         var pattern = /[0-9]+/g
         var ManagedClustersCount = fullText.match(pattern)
         var expectedSearchClusterCount = Number(ManagedClustersCount)
 
-        // local-cluster is default show in some filter conditions
-        if (extraCluster == 'has_local-cluster') {
-          expectedSearchClusterCount = expectedSearchClusterCount + 1
-        }
+        // // local-cluster is default show in some filter conditions
+        // if (extraCluster == 'has_local-cluster') {
+        //   expectedSearchClusterCount = expectedSearchClusterCount + 1
+        // }
```

Comment:
Commenting this out for now. I believed that this was fixed; however, currently, this will return the incorrect cluster count.